src/content/community/releases/_index.en.md

```diff
@@ -3,6 +3,35 @@ date = 2020-08-24T11:35:16+02:00
 title = "Releases"
 weight = 40
 +++
+<!-- markdownlint-disable no-duplicate-heading/no-duplicate-header -->
+## v0.13.0
+
+### New features
+
+* All Submariner container images are now available for x86-64 and ARM64 architectures.
+* Support was added in `subctl cloud prepare` to deploy Submariner on OpenShift on Microsoft Azure. This automatically configures the
+underlying Azure cloud infrastructure to meet Submariner's prerequisites.
+* Added a new, more robust, support for connecting clusters that use the OVNKubernetes CNI plugin in non-Globalnet deployments. Note that
+OVNKubernetes requires the OVN NorthBound DB version to be 6.1.0 or above and older versions are not supported. Also note that the minimum
+supported OpenShift Container Platform (OCP) version is 4.11.
+* Added support for connecting to Kubernetes headless Services without Pod label selectors in Globalnet deployments. This is useful when you
+want to point a Service to another Service in a different namespace or external network. When endpoints are manually defined by the user,
+Submariner automatically routes the traffic and provides DNS resolution.
+* Added a new `subctl show brokers` command that displays information about the Submariner Brokers installed.
+* The `subctl diagnose` command was extended to verify inter-cluster connectivity when Submariner is deployed using a LoadBalancer Service.
+
+### Other changes
+
+* The `submariner-operator` namespace is labeled in accordance with _KEP-2579: Pod Security Admission Control_ (default in Kubernetes 1.24)
+to allow the privileged Pods required for Submariner's interactions with iptables.
```

Comment:
```suggestion
to allow the Pods to be privileged.
```

This was revised between after the GDoc on the PR. https://submariner.io/community/releases/#v0122